### PR TITLE
Refactoring to coffee

### PR DESCRIPTION
**Pull request by upatricck**
Changing hardcoded solar values, to make them dynamic so that they can be used for coffee.

#DONE
* Creating a function that generates values based on sensor values installed
* Changing the status card
* Changing the graphs
* Changing the csv export

### CODE REVIEW

File: seshdash/views.py [hunk 1]
```diff
@@ -1388,7 +1379,7 @@ def delete_report(request, report_id):
     return redirect(reverse('manage_reports', args=[site.id]))
 
 @login_required
```
Comment (jlark) [inline seshdash/views.py]: is this method secure? We know login is needed but what if a logged in users uses a form to POST with a site id he doesn't have permission to use?

Comment (upatricck) [inline seshdash/views.py]: No its not any logged in user can post and get the csv, I will fix it.

Comment (jlark) [inline seshdash/views.py]: has this been fixed?

**Comment by jlark** [comment]
Merging for now. Will create another ticket to fix CSV download bug.

**Pull request merged**